bootstrapvz/common/assets/init.d/expand-root

```diff
@@ -13,8 +13,19 @@
 prog=$(basename $0)
 logger="logger -t $prog"
 
-device_path="/dev/xvda"
+growpart=$(which growpart) || {
```

Comment:
We shouldn't use `which`. [Consider using `type` or `hash` instead](http://stackoverflow.com/a/677212/339505).